app/src/test/kotlin/com/intive/picover/parties/repository/PartiesRepositoryTest.kt

```diff
@@ -80,5 +94,35 @@ class PartiesRepositoryTest : ShouldSpec(
 				tested.partyById("1").first()
 			}
 		}
+
+		should("add party with correct id") {
+			val childDatabaseReference: DatabaseReference = mockk {
+				every { key } returns newKey
+				coEvery { setValue(newPartyRemote).await() } returns mockk()
+			}
+			every { databaseReference.push() } returns childDatabaseReference
+			every { partyRemote.copy(newKey) } returns newPartyRemote
+
+			val result = tested.addParty(partyRemote)
+
+			assertSoftly {
+				coVerify {
+					childDatabaseReference.setValue(newPartyRemote).await()
+				}
+				result.shouldBeSuccess()
+			}
+		}
+
+		should("throw an exception when setValue fails") {
+			val errorMessage = "An exception"
+			every { databaseReference.push() } throws Exception(errorMessage)
+
+			val result = tested.addParty(partyRemote)
+
+			assertSoftly {
```

Comment:
please put result in method :) check the usage of this across the project :)

Reply:
@marcinWisnia Fixed